src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/SendCustomTx/index.tsx

```diff
@@ -224,6 +224,7 @@ const SendCustomTx: React.FC<Props> = ({ initialValues, onClose, onNext, contrac
                       placeholder="Data (hex encoded)*"
                       text="Data (hex encoded)*"
                       type="text"
+                      validate={mustBeHexData}
```

Comment:
Is this related to disabling submit buttons?
I don't mind it being in this PR but we probably should mention it for QA.